.github/workflows/pull_request_previews.yml

```diff
@@ -1,31 +1,33 @@
+# Synchronize pull requests to https://wptpr.live
+#
+# This task performs a potentially dangerous action (uploading code to
+# https://wptpr.live), so we restrict it to running on the base commit only
+# ('pull_request_target') so that malicious actors cannot modify the workflow
+# or scripts to force an upload.
 name: pr-preview-sync
 on:
-  schedule:
-    - cron: '*/5 * * * *'
+  # TEMPORARY FOR TESTING
+  #  pull_request_target:
+  #    types: [opened, synchronize, reopened, closed, labeled, unlabeled]
+  pull_request:
+    types: [opened, synchronize, reopened, closed, labeled, unlabeled]
 jobs:
   update-pr-preview:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
       with:
         fetch-depth: 1
     - name: Install dependency
       run: pip install requests
-    - name: Synchronize state
+    - name: Deploy PR
       # Use a conditional step instead of a conditional job to work around #20700.
       if: github.repository == 'web-platform-tests/wpt'
       run:
         ./tools/ci/pr_preview.py
           --host https://api.github.com
           --github-project web-platform-tests/wpt
-          synchronize
-          --window 480
+          --target https://wptpr.live
+          --timeout 600
       env:
-        # This Workflow must trigger further workflows. The GitHub-provided
-        # `GITHUB_TOKEN` secret is incapable of doing this [1], so a
-        # user-generated token must be specified instead. This token requires
-        # the "repo" scope, and is should be stored as a Secret named
-        # "DEPLOY_TOKEN" in this GitHub project.
-        #
-        # [1] https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows
```

Comment:
> as the default GITHUB_TOKEN when running from a fork

I thought the point of `pull_request_target` was that it ran in the context of the base repository not the fork, and thus the `GITHUB_TOKEN` should be read/write?

From https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/ (emphasis mine):

> In order to solve this, we’ve added a new pull_request_target event, which behaves in an almost identical way to the pull_request event with the same set of filters and payload. However, instead of running against the workflow and code from the merge commit, the event runs against the workflow and code from the base of the pull request. This means the workflow is running from a trusted source **and is given access to a read/write token** as well as secrets enabling the maintainer to safely comment on or label a pull request.


To be clear: it is my belief that using `GITHUB_TOKEN` for this workflow should work, and the only reason I am using `DEPLOY_TOKEN` here is for testing, because I am also using `pull_request` for testing (because `pull_request_target` of course wouldn't fire!). I apologise, I referenced this by saying 'the second commit should be reverted' but I should have been more explicit about what the changes in it were.

Reply:
Oh sorry I didn't realize that, so yeah using `GITHUB_TOKEN` LGTM.